saleor/graphql/attribute/mutations/attribute_value_create.py

```diff
@@ -34,10 +39,19 @@ class Arguments:
         )
 
     class Meta:
+        auto_permission_message = False
         model = models.AttributeValue
         object_type = AttributeValue
-        description = "Creates a value for an attribute."
-        permissions = (ProductPermissions.MANAGE_PRODUCTS,)
+        description = (
+            "Creates a value for an attribute.\n\n"
+            "Depending on the attribute type, "
+            "it requires different permissions to create:\n"
+            f"`PRODUCT_TYPE` requires {CREATE_PRODUCT_TYPE_PERMISSIONS_TEXT} "
```

Comment:
IMHO, I would fully drop the logic that prepares the permissions text and replace it with normal text. It doesn't make sense to have any automatic logic here. We are not going to extend/change this permissions in the future (except clean up in 4.0).
Also your message is different than default one for permission that we use.  We should have the same sentence as we have in `saleor.permission.utils.message_one_of_permissions_required`